core/jazz_environment-event-handler/index.js

```diff
@@ -571,7 +565,7 @@ function triggerBuildJob(payload, serviceDetails, configData) {
     var type;
     if (payload.service === 'ui' && payload.domain === 'jazz') {
       type = 'ui';
-      buildQuery = `/build?token=${configData.JOB_TOKEN}`;
+      buildQuery = `/buildWithParameters?token=${configData.JOB_TOKEN}&scm_branch=${payload.physical_id}`;
```

Comment:
Did Jenkins interface change from `build` to `buildWithParameters`?

Reply:
Yes, we need to change build to buildWithParameters as we need to pass the branch name.